### PR TITLE
Chronograf Install Module

### PR DESCRIPTION
**Pull request by tonerdo**
This PR contains the following:

`install-chronograf`: A shell script to install the chronograf binary
`chronograf-ami`: A Packer sample template to build Amazon AMIs and docker images with Chronograf installed

### CODE REVIEW

File: examples/chronograf-ami/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+[install-chronograf](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/install-chronograf)
+modules with [Packer](https://www.packer.io/) to create [Amazon Machine 
+Images (AMIs)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) that have 
+[Chronograf](https://www.influxdata.com/time-series-platform/chronograf/), and its dependencies installed on top of:
+ 
+1. Ubuntu 18.04
+1. Amazon Linux 2
+
+Chronograf is usually installed as a collection agent on your application's EC2 instance(s).
+This Chronograf AMI is only useful for standing up Chronograf to pull remote data or accept data from a remote service,
```
Comment (robmorgan) [inline examples/chronograf-ami/README.md]:
```suggestion
This Chronograf AMI is only useful for starting up Chronograf to pull remote data or accept data from a remote service,
```
?

File: examples/chronograf-ami/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "type": "shell",
+    "inline": [
+      "git clone --branch <MODULE_VERSION> https://github.com/gruntwork-io/terraform-aws-influx.git /tmp/terraform-aws-influx",
+      "/tmp/terraform-aws-influx/modules/install-chronograf/install-chronograf --version {{user `version`}}"
+    ],
+    "pause_before": "30s"
+  }]
+}
+```
+
```
Comment (robmorgan) [inline examples/chronograf-ami/README.md]: 👍

File: examples/chronograf-ami/config/chronograf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+HOST=<__HOST__>
+PORT=<__PORT__>
```
Comment (robmorgan) [inline examples/chronograf-ami/config/chronograf]: is there any point making sure these files have a newline at the end of them?

Comment (tonerdo) [reply examples/chronograf-ami/config/chronograf]: During development, Chronograf weirdly complained about the newline so I removed it. Will do more tests as I continue on to the run module

Comment (robmorgan) [reply examples/chronograf-ami/config/chronograf]: Ok, I was wondering 👍

**Pull request merged**